elements/rh-pagination/rh-pagination.css

```diff
@@ -1,28 +1,41 @@
 :host {
+  --_stepper-size: 48px;
+
   display: block;
   min-height: 4em;
 }
 
+:host([size="sm"]) {
+  --_stepper-size: 34px;
```

Comment:
@adamjohnson, can the stepper size for the small variant be changed to `32px`, so it matches with our length token sizes. This can use `var(--rh-length-2xl, 32px)` if you'd like.

Reply:
Resolved in [47c3440](https://github.com/RedHat-UX/red-hat-design-system/pull/1587/commits/47c3440d6980140563b64f6b6903459ee08d36e5).